### DISCUSSION TITLE
Billing Questions- Purchased and a total mess

### DISCUSSION THREAD

**Discussion by Rusty-Gopher-Phoenix-Nest** [primefaces/community#2995] [PrimeNG Blocks]
Hello, is there any "real" support from this company other than an e-mail address or this and Discord? 
I went to purchase what I thought was an upgrade to my Blocks which it gave me a code for. It then created a new account and charged me the full price and never asked for the coupon. Well it turns out all these areas have different accounts and passwords so it was different from the account I originally purchased this under!! 
I need to be able to fix this and associate it to the right account. 
I also finally find the "correct" Blocks page and the new Tailwind version for Angular isn't even out yet!!  
I could swear I got a notification to update otherwise I wouldn't have... anyway right now $209 wasted!!!
I would have thought there would be immediate support for this kind of thing... ugh

**Comment by thminuth** [comment]
the same happend to me. I have no access to my blocks any more and the code they gave for getting the new ones is useless. I changed to ngzorre

**Comment by Rusty-Gopher-Phoenix-Nest** [comment]
As a follow up they did completely fix this for me and refund what I paid. So thanks!!
